gc/gc-base/src/main/java/org/projectnessie/versioned/persist/gc/GCResult.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2020 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.versioned.persist.gc;
+
+import java.util.Map;
+
+public class GCResult<GC_CONTENT_VALUES extends ExpiredContentValues> {
```

Comment:
why wrap the map in a class? Especially when we expose the map for modification anyways?

Reply:
GCResult represents the final interface output. So, if we change underlying implementation [say instead of map to list], interface won't change.

I have used from @snazy's code, I thought this is Facade pattern.